### PR TITLE
task_1

### PR DESCRIPTION
**Pull request by ZiaurRehmanmalhi**

### CODE REVIEW

File: tasks/test_1.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from selenium import webdriver
+import time
+
+driver = webdriver.Chrome(executable_path="chromedriver")
+driver.get("https://www.youtube.com/@zusmani78/videos")
+
+scroll_distance = 2000
+i = 0
+
+while True:
```
Comment (Hamma111) [inline tasks/test_1.py:9]: so the loop will keep running infinitely. it will never stop. i am expecting better solutions from you.

File: chromedriver
Comment (Hamma111) [inline chromedriver:1]: always put the unwanted files in .gitignore.

**Review by Hamma111 — CHANGES_REQUESTED** [review]

File: chromedriver
Comment (ZiaurRehmanmalhi) [reply chromedriver:1]: ok

File: tasks/test_1.py (hunk 1 above)
Comment (ZiaurRehmanmalhi) [reply tasks/test_1.py:9]: ok

**Review by Hamma111 — APPROVED** [review]

**Pull request merged**